engine/evd.ml

```diff
@@ -463,6 +463,26 @@ type evar_map = {
   extras : Store.t;
 }
 
+let evar_map_info map =
+  let p name size =
+    if size > 0 then
+      Printf.printf "%s size = %d\n" name size in
```

Comment:
Please don't output to stdout from the engine.

Reply:
> Please don't output to stdout from the engine.

A bit early to comment: this is only a draft PR of some debugging code.  We don't know yet whether it's useful enough to keep around permanently.  If it is, we could certainly call it something like `debug_evar_map_info` to make its status more clear.  An alternative would be to expose the details of the `evar_map` type, which seems less desirable.

Reply:
I think @ejgallego was just saying something like "use msg_info or msg_debug or whatever the functions are called, rather than printing to stdout directly"?

Reply:
Indeed, at the very least you'd like to do what @JasonGross said [the point of submitting a draft PR is to gather feedback]

By the way in my interim branch, I have extensive support for stats, but indeed they are handled at the Document-manager level for now. The most convenient API would be to expose a `evar_stats` record.